README.md

```diff
@@ -13,58 +16,15 @@ Ie: `yum install nrjmx`
 
 You can download and decompress the Java executable as well from the [downloads url](http://download.newrelic.com/infrastructure_agent/binaries/linux/noarch/)
 
-### nri-jmx relation
+### New Relic JMX integration (nri-jmx)
 
-`nrjmx` is *not* bundled within the `nri-jmx` package. But, it's declared as a dependency. 
+`nrjmx` is *not* bundled within the `nri-jmx` package. But, it's declared as a dependency.
 
 So while installing `nri-jmx` if you have `nrjmx` already installed it keeps the installed version, otherwise it'll try to get the latest `nrjmx` release.
 
+## Getting Started
 
-## Custom Build
```

Comment:
@ramonguiu Can we move this information to a new .md file in a /docs/ subfolder?